src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -514,6 +515,26 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 		return overrides;
 	}
 
+	private registerDefaultOverride(): void {
+		this._register(this.editorOverrideService.registerContributionPoint(
+			'*',
+			priorityToRank(ContributedEditorPriority.builtin),
+			{
+				id: DEFAULT_EDITOR_ASSOCIATION.id,
+				label: DEFAULT_EDITOR_ASSOCIATION.displayName,
+				detail: DEFAULT_EDITOR_ASSOCIATION.providerDisplayName,
+				active: (currentEditor) => this.fileEditorInputFactory.isFileEditorInput(currentEditor) && isEqual(currentEditor.resource, this.activeEditor?.resource),
+				priority: ContributedEditorPriority.builtin,
+			},
+			{},
+			(resource, editorID, options, group) => {
+				return { editor: this.createEditorInput({ resource }) };
```

Comment:
What about the `options`?

Reply:
If the options aren't passed back then we use the ones that were passed in which I assume is what we want here